autogpt_platform/backend/backend/blocks/ayrshare/_api.py

```diff
@@ -0,0 +1,482 @@
+from __future__ import annotations
+
+import json
+import logging
+from dataclasses import dataclass
+from enum import Enum
+from typing import Any, Dict, List, Optional, Union
```

Comment:
```suggestion
from typing import Any, Optional
```
pls use `dict`, `list`, `|` instead